Add a comparison leaderboard

p-urple opened the pull request:
compares the user's upvotes/downvotes to the server's upvotes/downvotes and then the average.
Formula:
(server_average_upvotes * (user_total_upvotes / server_total_upvotes)) / (server_average_downvotes * (user_total_downvotes / server_total_downvotes)

p-urple committed:
Add a comparison leaderboard

compares the user's upvotes/downvotes to the server's upvotes/downvotes and then the average.
Formula:
(server_average_upvotes * (user_total_upvotes / server_total_upvotes)) / (server_average_downvotes * (user_total_downvotes / server_total_downvotes)

itsmehemant123 commented on leaderboard.py:
```diff
@@ -455,13 +483,29 @@ def generate_memer_board(self, ctx, method, lim, span):
-        
+        elif (method == 'compare_up'):
+            memers = self.session.query(func.sum(Message.rx1_count), func.sum(Message.rx2_count)).filter(
+                Message.server_id == db_server.id, Message.created_at > start_date, Message.user_id == db_user.id).group_by(Message.user_id).all()
```
you don't have the `db_user` object, so you'll have to get it first before using it.

p-urple committed:
fixed sorting of members

you pointed out my mistake at interpreting your code because I was tired so I fixed it to actually sort it properly (in other words I didn't take the time to understand how it sorted values until you told me I was wrong because I'm lazy)
